packages/apps-sdk/src/__tests__/listener.spec.ts

```diff
@@ -0,0 +1,71 @@
+/* eslint no-underscore-dangle: 0 */
+
+import Listener from '../listener';
+
+describe('Listener', () => {
+  let listener: Listener;
+
+  beforeEach(() => {
+    listener = new Listener();
+  });
+
+  describe('method:addListener', () => {
+    it('should regiter listeners when calling addListeners', () => {
+      const mockHandler1 = jest.fn();
+      const mockHandler2 = jest.fn();
+      listener.addListener('type-1', mockHandler1);
+      listener.addListener('type-2', mockHandler2);
+
+      listener.fire('type-1');
+      listener.fire('type-2');
+      expect(mockHandler1).toHaveBeenCalled();
+      expect(mockHandler2).toHaveBeenCalled();
+    });
+
+    it('should call event listeners on event', () => {
+      const received: string[] = [];
+
+      listener.addListener('type-1', (n: string) => received.push(n));
```

Comment:
Why `n` is a string ? as I can see you fire this event with an object `listener.fire('type-1', { eventName: 'test1' })`.

Reply:
And may be I am wrong but I suppose that the previous test checks the same thing, doesn't it ?